README.md

```diff
@@ -3,24 +3,27 @@ Visit the [website](http://www.aurorabuilder.com "Aurora Website") for more info
 
 # Aurora Legacy Project
 The goal of Aurora Legacy is to continually support Aurora Builder app with updates to content files.  <br>
+[Project Discord Server](https://discord.gg/3n5uakXT2a "Project Discord Server")  <br>
```

Comment:
Having it as just `Project Discord Server` on its own line feels weird, so i added a bit of a description.
```suggestion
[Project's Discord Server](https://discord.gg/3n5uakXT2a "Project Discord Server"), where you can find the announcements, discuss relevant news and receive help with your homebrew files.  <br>
```